src/backend/distributed/sql/udfs/drop_old_time_partitions/latest.sql

```diff
@@ -0,0 +1,57 @@
+-- heavily inspired by the procedure alter_old_partitions_set_access_method
+CREATE OR REPLACE PROCEDURE pg_catalog.drop_old_time_partitions(
+	parent_table_name regclass,
```

Comment:
we use table_name in create_time_partitions and parent_table_name in drop_old_time_partitions. I think either one is fine, but they should be the same.